doc/sphinx/source/input.rst

```diff
@@ -132,17 +132,18 @@ To use this functionality, users need to provide a path for the ``native6`` proj
     datasets:
     - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3, start_year: 1990, end_year: 1990}
 
-Currently, the native6 project only supports ERA5 and ERA5-Land data in the format defined in the `config-developer file <https://github.com/ESMValGroup/ESMValCore/blob/a9312a7d5be4fa3aac55c0b2ef089c6b4e1a61a9/esmvalcore/config-developer.yml#L191-L201>`_. ERA5 data can be downloaded using `era5cli <https://era5cli.readthedocs.io>`_ To support other datasets as well, we need to make it possible to have a dataset specific DRS. This is still on the horizon.
+More examples can be found in the example diagnostics ``ERA5_native6`` in the recipe `examples/recipe_check_obs.yml <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_check_obs.yml>`_.
+Currently, the native6 project only supports ERA5 and ERA5-Land (hourly and monthly) data in the format defined in the `config-developer file <https://github.com/ESMValGroup/ESMValCore/blob/a9312a7d5be4fa3aac55c0b2ef089c6b4e1a61a9/esmvalcore/config-developer.yml#L191-L201>`_. ERA5 data can be downloaded using `era5cli <https://era5cli.readthedocs.io>`_ To support other datasets as well, we need to make it possible to have a dataset specific DRS. This is still on the horizon.
```

Comment:
MSWEP is also supported, this is documented here (a rather hard to find choice for a location to add documentation): https://docs.esmvaltool.org/projects/esmvalcore/en/latest/develop/fixing_data.html#natively-supported-non-cmip-datasets

Dataset specific DRS is no longer needed since https://github.com/ESMValGroup/ESMValCore/pull/645 was merged and the idea has been abandoned, see discussion in https://github.com/ESMValGroup/ESMValCore/issues/494

Would it be possible to rewrite this section so it is a bit less specific to ERA5? I noticed that in the table below the ERA5-Land dataset is missing the indication that it is supported natively and MSWEP is completely missing. If it's not too much work, would you be willing to update that too in this pull request?

Reply:
Thanks. The text is now revised. Is it better? Also, the table and the text below it are updated.

Reply:
Much better, thanks!